Apps/AdminWebClient/src/ClientApp/src/models/adminCommunication.ts

```diff
@@ -40,6 +45,9 @@ export default interface Communication {
     // Gets or sets the communication status code: new or processed
     communicationStatusCode: string;
 
+    // Gets or sets the communication publishing status code: Draft or Publish
+    publishingStatusCode: string;
```

Comment:
should this be an enum too?

Reply:
no, I don't think so as the communicationStatusCode above is also a string right?

Reply:
if both can be changed that might be even better. That's -if-

Reply:
Sounds good, I will look at our other areas to see if we can do this easily here, if not I will create a ticket to capture this refactoring later then.

Reply:
As we don't introduce the new Publishing Status, I prefer to defer this refactoring to another task then.